### DISCUSSION TITLE
1-Click Quick Request Feature

### DISCUSSION THREAD

**Discussion by teeesss** [seerr-team/seerr#631] [Ideas]
Currently when you click Request a pop-up comes up, loads all seasons for a show, i.e. S01, S02, S03, the quality profile and the tags and for a movie just the quality profile and tags.

Would like additional buttons for quick 1-Click requests for both All Seasons and one for Current/Future Season.
It would be a 1-Click option for either but could be configurable as well as noted below.
The layout could be All, Current, Request.

**All**
Would default to downloading all seasons in the series and take default quality profile and tag.

**Current**
Would default to downloading current season or upcoming season(s), taking the default quality profile and tag.

**Request**
This stays the same.

**Configurable option in the settings**
This would be a setting you could configure for the new All or Current options.
Allow it to be configurable so there are options to either make it 1-Click or more specific depending on the user.
Maybe I wanted a 1-Click option only or maybe I wanted to just be asked to confirm the quality profile or just the tags.

95% of the time, I'm clicking Request and I am using the defaults so having those other options available would be beneficial and save time.

Thanks!

**Comment by damientkyt** [comment]
With the custom rules, I think we should be able to have click once to request using the custom rules and select all seasons, including previously un-selected seasons.

**Reply by kratoz29** [reply]
I just tried this and sadly, no luck for me, I still need to hit twice the request button :/

**Comment by chazzawalla** [comment]
Agreed, another option would be to give us an option in settings where TV requests default to no seasons, all seasons, or current+future seasons. This would allow series requests to be basically one to two taps, just like it is for movies.

The current method when on mobile requires stretching one’s thumb all the way up to the top far corner of the screen to select seasons, then back down to submit the request. Not a polished experience.
